examples/todos/src/components/App.spec.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from './App';
+
+const setup = () => {
+  const wrapper = shallow(<App/>);
+  return { wrapper };
+};
+
+describe('App component', () => {
+  it('Should render App component', () => {
+    const { wrapper } = setup();
+    expect(wrapper).toMatchSnapshot();
```

Comment:
In case we have complicated setup so we need extract to `setup` method. However, in this case, it's quite simple, so we can refactor code better such as:
```
const wrapper = shallow(<App/>);
expect(wrapper).toMatchSnapshot();
```